force-app/main/default/aura/TaskKanban/TaskKanbanController.js

```diff
@@ -0,0 +1,104 @@
+                });  
+            }
+        }
+	},
+	onContactListChangeHandler : function(component,event,helper)
+	{
+        let cId = component.find("contactSelected").get("v.value");
+       
+		helper.loadTaskOfContact(component,cId).then(result => {
+            component.set("v.selectedContact",helper.getContactSelectChanged(component,cId));
+            helper.parseContactTaskIntoLists(component,result);
+        }).catch(error => {
+           
+        });         
+	},
+	openNewTaskCardHandler : function(component, event, helper){
+        
+        let cId = component.find("contactSelected");
+        
+        if(component.get("v.recordId") !== undefined || (cId !== undefined  && cId.get("v.value") !== "-1" ))
+        {
+            helper.handleNewTaskModalOpenClose(component,event);
+        }else 
+        {
+             helper.displayToast(component,"error","Alert!","Please select a contact to continue");
+        }
+    },
+	newTaskHandler : function(component,event,helper)
+	{
+		helper.saveNewTask(component,event,component.get("v.selectedContact")["Id"]).then(result => {
+                helper.displayToast(component,"success","Success!","Task Created Successfully!");
+                
+            	helper.loadTaskOfContact(component,component.get("v.selectedContact")["Id"]).then(result => {
+                component.set("v.tasksOfContact",result);
+                helper.parseContactTaskIntoLists(component,result);
+                
+            }).catch(error => {
+                helper.displayToast(component,"error","error!",error);
+            }); 
+
+        }).catch(error => {
+            helper.displayToast(component,"error","error!",error);
+		});   
+	},
+	closeTaskHandler : function(component,event,helper)
+	{
+        helper.closeTask(component,event).then(result => {
+           		helper.displayToast(component,"success","Success!","Task Completed Successfully!");
+                
+            helper.loadTaskOfContact(component,component.get("v.selectedContact")["Id"]).then(result => {
+                component.set("v.tasksOfContact",result);
+                helper.parseContactTaskIntoLists(component,result);
+                
+            }).catch(error => {
+                helper.displayToast(component,"error","error!",error);
+            }); 
+
+        }).catch(error => {
+			
+		});            
+	},
+	deleteTaskHandler  : function(component,event,helper)
+	{
+        helper.deleteTaskHandler(component,event).then(result => {
+            helper.displayToast(component,"success","Success!","Task Deleted Successfully!");
+            helper.loadTaskOfContact(component,component.get("v.selectedContact")["Id"]).then(result => {
+                    component.set("v.tasksOfContact",result);
+                    helper.parseContactTaskIntoLists(component,result);
+                    
+                }).catch(error => {
+                    helper.displayToast(component,"error","error!",error);
```

Comment:
this is the kind of error messages that Salesforce love 😆

Reply:
Agreed ! Added more information on the message of the toast, will add changes them to the next commit